source/BackwardTaintEnvironment.cpp

```diff
@@ -25,7 +25,9 @@ BackwardTaintEnvironment BackwardTaintEnvironment::initial(
     MethodContext& context) {
   auto taint = TaintEnvironment::bottom();
 
-  if (!context.method()->is_static()) {
+  const Method& method = *context.method();
```

Comment:
I would keep using `method` as a pointer, to be consistent with the rest of the codebase.
```suggestion
  const Method* method = context.method();
```